src/MongoDB/Session.c

```diff
@@ -408,15 +426,16 @@ ZEND_END_ARG_INFO()
 
 static zend_function_entry php_phongo_session_me[] = {
 	/* clang-format off */
+	PHP_ME(Session, abortTransaction, ai_Session_void, ZEND_ACC_PUBLIC | ZEND_ACC_FINAL)
 	PHP_ME(Session, advanceClusterTime, ai_Session_advanceClusterTime, ZEND_ACC_PUBLIC | ZEND_ACC_FINAL)
 	PHP_ME(Session, advanceOperationTime, ai_Session_advanceOperationTime, ZEND_ACC_PUBLIC | ZEND_ACC_FINAL)
+	PHP_ME(Session, commitTransaction, ai_Session_void, ZEND_ACC_PUBLIC | ZEND_ACC_FINAL)
+	PHP_ME(Session, endSession, ai_Session_void, ZEND_ACC_PUBLIC | ZEND_ACC_FINAL)
+	PHP_ME(Session, isInTransaction, ai_Session_void, ZEND_ACC_PUBLIC | ZEND_ACC_FINAL)
```

Comment:
Since it looks like you're alphabetizing these, `isInTransaction` should go after the `get` methods.

Reply:
Yeah, I had alphabetised them in the first commit, ... and then put the new one in the wrong place :-)